src/util/cache/cloneIntoCache.ts

```diff
@@ -18,17 +16,22 @@ const git = simpleGit();
  * @returns void, or throws an error if the repository could not be cloned.
  */
 export default function cloneIntoCache(
-  type: CacheBucket,
+  bucket: CacheBucket,
   itemPath: CacheItemPath
 ) {
   return async ({ repository, checkout }: GitCloneOptions): Promise<void> => {
-    // @TODO: Eventually, this needs to support being able to clone different copies of the
-    // same system for different projects.
-    const destination = join(CACHE_DIR, type, ...itemPath);
+    const destination = getCachedItemPath(bucket, itemPath);
     const parentDir = dirname(destination);
+    let git = simpleGit();
```

Comment:
This is using `let` because if the cached item already exists, we have to re-instantiate simpleGit and pass in the path to the repo's directory.